javascript/modules/client/test/integration/client.test.ts

```diff
@@ -450,6 +504,70 @@ describe("Client", () => {
     });
   });
   describe("Action decoders", () => {
+    it("Should decode an encoded grant action", async () => {
+      const context = new Context(contextParamsLocalChain);
+      const client = new Client(context);
+
+      const grantParams: IPermissionParams = {
+        who: "0x1234567890123456789012345678901234567890",
+        where: "0x1234567890123456789012345678901234567890",
+        permission: DaoPermissions.UPGRADE_PERMISSION,
+      };
+
+      const grantAction = await client.encoding.grantAction(
+        grantParams,
+      );
+      const decodedGrantParams: IPermissionParams = client.decoding
+        .grantAction(grantAction.data);
+
+      expect(decodedGrantParams.permission).toBe(grantParams.permission);
```

Comment:
This should not be a hex string at all (permission ID). This should be the human readable permission name.